src/lib.rs

```diff
@@ -830,37 +893,45 @@ pub struct RenderPassDescriptor<'a, 'b> {
     pub depth_stencil_attachment: Option<RenderPassDepthStencilAttachmentDescriptor<'a>>,
 }
 
-/// A description of a buffer.
-pub type BufferDescriptor<'a> = wgt::BufferDescriptor<Option<&'a str>>;
+// The underlying types are also exported so that documentation shows up for them
+
+pub use wgt::BufferDescriptor as BufferDescriptorBase;
```

Comment:
The problem is that if you build just wgpu's documentation, you don't get a link to the wgt type as those docs weren't ever built. Additionally, even when all docs are built (such as on docs.rs), I feel like we shouldn't be linking to wgpu-types' doc page because that would break doc links for these types, and people will likely get confused as to where all their types went.